src/NHibernate.Test/Futures/LinqFutureFixture.cs

```diff
@@ -360,5 +360,36 @@ public void CanExecuteMultipleQueriesOnSameExpression()
 				}
 			}
 		}
+
+		[Test(Description = "https://github.com/nhibernate/nhibernate-core/issues/1387")]
+		public void ToFutureValueWithSumReturnsResult()
```

Comment:
Since it is a `FutureValue` only test (not mixed with `Future`), it may be better placed in `LinqToFutureValueFixture`.

Reply:
We likely need to also check the behavior of future with empty sum, and same on nullables.